dev/com.ibm.ws.cdi.ejb_fat/fat/src/com/ibm/ws/cdi/ejb/apps/newTimer/MyCDIInterceptorBinding.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2015 IBM Corporation and others.
```

Comment:
I assume this "new" file has been copied without really changing anything?

If not the copyright date should be 2022.

Reply:
You assume correctly.